ast/ddl.go

```diff
@@ -667,15 +668,16 @@ const (
 type AlterTableSpec struct {
 	node
 
-	Tp            AlterTableType
-	Name          string
-	Constraint    *Constraint
-	Options       []*TableOption
-	NewTable      *TableName
-	NewColumn     *ColumnDef
-	OldColumnName *ColumnName
-	Position      *ColumnPosition
-	LockType      LockType
+	Tp              AlterTableType
+	Name            string
+	Constraint      *Constraint
+	Options         []*TableOption
+	NewTable        *TableName
+	NewColumn       *ColumnDef
+	AppendedColumns []*ColumnDef
```

Comment:
Do we really need a `AppendedColumns` spec? I think we can extend the NewColumn to NewColumns